packages/squiggle-lang/__tests__/GenericDist/GenericDistSparkline_Test.res

```diff
@@ -0,0 +1,30 @@
+open Jest
+open Expect
+
+let normalDist: GenericDist_Types.genericDist = Symbolic(#Normal({mean: 5.0, stdev: 2.0}))
```

Comment:
What if, hear me out, a `Fixtures` file where these are constructed? factor out some redundant lines of code and share stuff between test files. 

It might go against my more prevailing philosophy which is that "_when test code is 0 IQ, the person reading the code for the purposes of figuring out what all the code does has the easiest time_"

Reply:
Anyways, I was thinking about asking you to make it look more like 
```rescript
let normalDist = ...
runTest("normal", normalDist, `▁▁▁▁▂▃▄▆▇██▇▆▄▃▂▁▁▁`)
```
and not frontload the file with the items, but decided I didn't care enough to trouble you with it.

Reply:
I like having fixtures, I really don't think this is much overhead.

Generally I think I prefer tests to be more elegant / easy to read (where one hopefully doesn't need to care much about some implementation details)

Reply:
I mean like:
Benefits: If the interface to GenericDist changes, you can make the change in one place
Downsides: It means the reader has to resolve what something means in their heads.

Frankly, I think it would be worth it, especially considering that the actual parameters are not really relevant to the test, just the fact that it's a particular type of distribution